tests/behavioural/features/steps/message_post.py

```diff
@@ -141,6 +142,55 @@ def step_impl(context):
                  'survey': 'survey'})
 
 
+# Scenario 12: When a message with the label of "Draft" is sent and another user is trying to send the same message return a 409
+@given('a draft message is posted')
+def step_impl(context):
+    if 'msg_id' in data:
+        del data['msg_id']
+
+    context.post_draft = app.test_client().post("http://localhost:5050/draft/save", data=json.dumps(data),
+                                                headers=headers)
+    #get etag from response using context
+    context.etag =  json.loads(context.post_draft.data)
+    headers['Etag'] = context.etag
```

Comment:
I dont think this is a valid Etag ? Shouldnt there be a function call / something that hashes the data ?